### DISCUSSION TITLE
SDM630 - Energy Total in Tasmota differs from device value

### DISCUSSION THREAD

**Discussion by lzimmermann** [arendst/Tasmota#19065] [Q&A]
Hi all,

I have the SDM639 Smart Meter and try reading the values with tasmota - that works all fine - thanks for your effort!

But I see an issue with the Energy Total value - in Tasmota it differs from the value the meter shows. Its much less than the devices one, but increases correctly. 
Is it possible to show the same value as in the meter device? 

So currently I see some 423KWh in Tasmuta and 645KWh in the meter.

**Comment by curzon01** [comment]
set [SO72](https://tasmota.github.io/docs/Commands/#setoption72)
